### PR TITLE
[CAM-12937] adding images

### PR DESCRIPTION
**Pull request by SharonWang1005**

### CODE REVIEW

File: Cloud Application Manager/Core Concepts/providers.md [hunk 1]
```diff
@@ -100,21 +100,21 @@ Cloud Application Manager integrates with many popular clouds. Here is a full li
...
+| [CenturyLink Cloud](../Deploying%20Anywhere/using-centurylink-cloud.md)                                               | ✓                      | ✓                           |
+| [CenturyLink DCC](../Deploying%20Anywhere/using-dcc.md)                                                               | ✓                      | ✓                           |
+| [CenturyLink Private Cloud on VMware Cloud Foundation](../../Deploying%20Anywhere/using-private-cloud-on-vmware-cloud-foundation.md) | ✓                      | ✓                           |
+| [Amazon Web Services](../Deploying%20Anywhere/using-your-aws-account.md)                                              | ✓                      | ✓                           |
+| Amazon Web Services GovCloud (Same documentation as [Amazon Web Services](../Deploying%20Anywhere/using-your-aws-account.md)) | ✓              | ✓                           |
+| [CloudStack](../Deploying%20Anywhere/using-cloudstack.md)                                                             | ✓                      | ✓                           |
```
Comment (jim-phillips) [inline Cloud Application Manager/Core Concepts/providers.md]: Please change all `%20` back to an actual space ` `. 
These links are to a relative file, not http links.  Even with http links, the system will handle the conversion, so no need for encoding.

**Comment by jim-phillips** [comment]
@SharonWang1005 Please fix links, and resolve conflicts.

**Comment by SharonWang1005** [comment]
deleted the %

**Comment by SharonWang1005** [comment]
@gsanchezu please review.

**Comment by gsanchezu** [comment]
Hey @SharonWang1005, please review the Travis results: https://travis-ci.org/CenturyLinkCloud/PublicKB/builds/614066052?utm_source=github_status&utm_medium=notification

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### Prerequisites
+
+* An active **Cloud Application Manager** account
+* Any infrastructure capable of running Cloud Application Manager Agent.
+
+
+### Create a Provider
+
+The preferred method to register any infrastructure in CAM, is to register it into a provider, instead of a stand-alone instance. To learn more about registering your servers as a stand-alone instance, please review the [Deploying Anywhere](./deploying-on-anyinfra.md)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]: I will remove this paragraph from here and add it to the `deploying-on-anyinfra.md` article (linking to this one instead)

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+To create a new provider, select **Compute Instances** from the Provider list and give it a name, then click on **Save**
+
+![centurylink-add-provider.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_1.png)
+
+**Note**: Unlike other provider types, this one does not require any synchronization process and it does not support deployment policies either, since there is no direct deployment support on it.
+
+### Register a Server
+You can launch boxes on any infrastructure by running the Cloud Application Manager agent. The agent is required to execute box variables and scripts and run lifecycle operations. Once the provider is created, users can register VMs or physical machines to this Provider by clicking **Self-Register Scripts**
+
+![register-a-server-1.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_2.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]: Please, use more meaningful alternate texts in all the article

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+There are 2 scripts for Windows and Linux OS. In this article, we use Windows server as an example:
+![register-a-server-2.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_3.png)
+
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]: Please, explain briefly about the proxy support in the code snippets (probably it is already explained in the self-registering article, but at least mention it here briefly)

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Copy the script and run on the Windows server we have on CLC to trigger the self-register process:
+![register-a-server-3.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_4.png)
+
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]: I would add that the provider activity log will show an activity when an instance is asking to be registered in the provider.

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+Once the script is completed, you can see the server registered:
+![register-a-server-4.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_5.png)
+
+![register-a-server-5.png](../../images/cloud-application-manager/deploying-anywhere/compute-instances/compute_instances_6.png)
+
+To learn more about Cloud application manager Self Register Instances, please review the [Self Register Instances](../Getting%20Started/self-register-instances.md)
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]:
```suggestion
To learn more about Cloud application manager Self Register Instances, please review the [Self Register Instances](../Getting Started/self-register-instances.md)
```
I think this is causing the Travis build to fail

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+"date": "11-05-2019",
```
Comment (jim-phillips) [inline Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]: missing opening `{{{`

**Comment by gsanchezu** [comment]
Hey @SharonWang1005, could you please address above comments?

File: Cloud Application Manager/Deploying Anywhere/using-compute-instances.md (hunk 2 above)
Comment (SharonWang1005) [reply Cloud Application Manager/Deploying Anywhere/using-compute-instances.md]: Sorry, you will do it or I should be the one doing it?

**Pull request merged**